runner.go

```diff
@@ -389,7 +389,19 @@ func (r *LocalRunner) NetworkCreator(network Network) func() error {
 		if Verbose {
 			log.Printf("Create network '%s'", network)
 		}
-		return r.Exec([]string{"network", "create", string(network)})
+		if _, err := r.Output([]string{"network", "create", string(network)}); err != nil {
+			if err, ok := err.(*exec.ExitError); ok {
+				if strings.TrimSpace(string(err.Stderr)) != fmt.Sprintf("Error response from daemon: network with name %s already exists", string(network)) {
```

Comment:
Maybe add a TODO that this is somewhat ugly but a consequence of invoking commands

Reply:
Yeah, that part isn't nice but sadly necessary only handle that special error.